### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Cosmosha**
Portfolio Project Desktop Version modified for the following sections;

-  Navbar
-  Works
-  About-Myself
-  Contact Me

The following Media Screens were targeted for content modification;

- Media Query screen target 768px
- Media Query screen target 1024px
- Media Query screen target 1440px

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -212,23 +218,22 @@ header {
...
-  background: #fff;
+  background-color: #fff;
   color: #5e6c84;
 }
 
 .action .btn:active {
-  background: #2230d2;
+  background-color: #2230d2;
   color: #fff;
 }
```
Comment (alexpuente0) [inline style.css]: - [ ] Hovering is not working properly on the desktop version. you can try to add the `:hover` and `:active` with the `btn:enabled` properties so you end up with fewer lines of code:
`.action .btn:disabled `
`.action .btn:enabled:hover `
`.action .btn:enabled:active `

**Review by alexpuente0 — CHANGES_REQUESTED** [review]
Hi @Cosmosha, 



You've done a great job so far!

In this case, however, there are still some issues that you need to work on to finish this project and advance to the next one, but you are almost there!

## Highlights
- You have very well documented code.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Try and complete your ReadMe File to make it more professional, as it currently has still some template text in it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Cosmosha) [reply style.css]: thank you for making time to review my codes. the necessary changes will be highly considered and worked on.

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @Cosmosha 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**